contracts/src/v0.8/workflow/test/WorkflowRegistry/WorkflowRegistry.getAllAuthorizedAddresses.t.sol

```diff
@@ -28,4 +28,15 @@ contract WorkflowRegistrygetAllAuthorizedAddresses is WorkflowRegistrySetup {
     assertEq(authorizedAddresses[0], s_authorizedAddress);
     assertEq(authorizedAddresses[1], s_unauthorizedAddress);
   }
+
+  function test_WhenTheRegistryIsLocked() external {
+    // Lock the registry
+    vm.prank(s_owner);
```

Comment:
since `prank` only works for the immediate next call, it's unclear who is calling `getAllAuthorizedAddresses`?
maybe do another prank to specifically test who calls getAllAuthorizedAddresses

same with other ones